pyrenew_hew/site_level_dynamics_model.py

```diff
@@ -0,0 +1,566 @@
+import jax
+import jax.numpy as jnp
+import numpyro
+import numpyro.distributions as dist
+import numpyro.distributions.transforms as transforms
+import pyrenew.transformation as transformation
+from numpyro.infer.reparam import LocScaleReparam
+from pyrenew.arrayutils import tile_until_n
+from pyrenew.convolve import compute_delay_ascertained_incidence
+from pyrenew.deterministic import DeterministicVariable
+from pyrenew.latent import (
+    InfectionInitializationProcess,
+    InfectionsWithFeedback,
+    InitializeInfectionsExponentialGrowth,
+)
+from pyrenew.metaclass import Model
+from pyrenew.observation import NegativeBinomialObservation
+from pyrenew.process import ARProcess, DifferencedProcess
+from pyrenew.randomvariable import DistributionalVariable, TransformedVariable
+
+from pyrenew_hew.utils import get_vl_trajectory
+
+
+class ww_site_level_dynamics_model(Model):  # numpydoc ignore=GL08
+    def __init__(
+        self,
+        state_pop,
+        unobs_time,
+        n_initialization_points,
+        i0_t_offset,
+        log_r_t_first_obs_rv,
+        autoreg_rt_rv,
+        eta_sd_rv,
+        autoreg_rt_subpop_rv,
+        sigma_rt_rv,
+        i_first_obs_over_n_rv,
+        sigma_i_first_obs_rv,
+        sigma_initial_exp_growth_rate_rv,
+        mean_initial_exp_growth_rate_rv,
+        offset_ref_logit_i_first_obs_rv,
+        offset_ref_initial_exp_growth_rate_rv,
+        offset_ref_log_r_t_rv,
+        generation_interval_pmf_rv,
+        infection_feedback_strength_rv,
+        infection_feedback_pmf_rv,
+        p_hosp_mean_rv,
+        p_hosp_w_sd_rv,
+        autoreg_p_hosp_rv,
+        hosp_wday_effect_rv,
+        inf_to_hosp_rv,
+        phi_rv,
+        hosp_times,
+        pop_fraction,
+        n_subpops,
+        t_peak_rv=None,
+        dur_shed_after_peak_rv=None,
+        n_ww_lab_sites=None,
+        max_shed_interval=None,
+        log10_g_rv=None,
+        mode_sigma_ww_site_rv=None,
+        sd_log_sigma_ww_site_rv=None,
+        ww_site_mod_sd_rv=None,
+        ww_ml_produced_per_day=None,
+        ww_uncensored=None,
+        ww_censored=None,
+        ww_sampled_lab_sites=None,
+        ww_sampled_subpops=None,
+        ww_sampled_times=None,
+        ww_log_lod=None,
+        lab_site_to_subpop_map=None,
+        max_ww_sampled_days=None,
+        include_ww=0,
```

Comment:
```suggestion
        include_ww=False,
```

Reply:
changed in https://github.com/CDCgov/pyrenew-hew/pull/49/commits/259e895ce55a0248dca4ffd590dc418dfb172dc9